src/tsickle_compiler_host.ts

```diff
@@ -124,7 +124,8 @@ export class TsickleCompilerHost implements ts.CompilerHost {
       fileName: string, languageVersion: ts.ScriptTarget,
       onError?: (message: string) => void): ts.SourceFile {
     if (this.runConfiguration === undefined || this.runConfiguration.pass === Pass.NONE) {
-      const sourceFile = this.delegate.getSourceFile(fileName, languageVersion, onError);
+      const sourceFile: ts.SourceFile =
```

Comment:
Leftover from when I was trying to annotate as ts.SourceFile | undefined.  Removed.